mqtt/client/mqpub-static.sh

```diff
@@ -51,4 +51,6 @@ then
     do
         $PUBBIN -h $mqtthost $auth -t $topic/port$i/lock/\$settable -m "true" -r
     done
-fi
\ No newline at end of file
+fi
+
+$PUBBIN -h $mqtthost $auth -t $topic/\$online -m "true" -r
```

Comment:
I checked here https://github.com/homieiot/convention#device-behavior

Shouldn't it be `$state=lost` instead of `$online=false` ???

Reply:
Mentioned this in my reply to a previous comment but adding it here for clarity as well:

As far as the $state property... this is one of the things that changed apparently between version 2 (https://github.com/homieiot/convention/tree/v2.0.1#device-properties) and version 3 (https://github.com/homieiot/convention/tree/v3.0.0#device-behavior) of the homie spec... in version 2 it is $online (true/false) and version 3 it is $state (init/ready/lost etc). Since the mpower script is reporting version 2.x when it starts up I chose $online

Reply:
right! Will change that once I upgrade to v3.